package-lock.json

```diff
@@ -7858,6 +7860,23 @@
 				"webpack-bundle-analyzer": "^3.3.2",
 				"webpack-cli": "^3.1.2",
 				"webpack-livereload-plugin": "^2.2.0"
+			},
+			"dependencies": {
```

Comment:
We will have to update how this gets resolved to avoid having `node_modules` inside `packages/scripts`. The easiest way to hoist it is to install the dependency as described in https://github.com/WordPress/gutenberg/tree/master/packages#adding-new-dependencies.

At this moment you can also install it in the root folder and remove afterwards.